src/components/Button/Button.js

```diff
@@ -172,4 +172,12 @@ Button.defaultProps = {
   kind: 'primary',
 };
 
-export default Button;
+export default (!breakingChangesX
+  ? Button
+  : (() => {
```

Comment:
Are these IIFEs so that the inner block is marked as PURE and can be stripped?

Reply:
This IIFE is merely for better variable scoping, I'm open for other approaches.

Reply:
I think it makes sense mostly for the fact that I think it'll all be stripped out in a minified build if `breakingChangesX` is false, right? Would be great to keep it around just for that.

Reply:
Cool - Let's keep the IIFE here!